frontend/pages/beta/model/[modelId]/access-request/schema.tsx

```diff
@@ -22,6 +23,11 @@ export default function NewSchemaSelection() {
   const activeSchemas = useMemo(() => schemas.filter((schema) => schema.active), [schemas])
   const inactiveSchemas = useMemo(() => schemas.filter((schema) => !schema.active), [schemas])
 
+  const error = MultipleErrorWrapper(`Unable to fetch schemas`, {
+    isSchemasError,
+  })
+  if (error) return error
+
   if (isSchemasError) {
```

Comment:
You check to see whether there is a `isSchemasError` here.  Why are you checking for it again three lines up?